wagtail/admin/tests/test_templatetags.py

```diff
@@ -1084,3 +1086,139 @@ def test_light_mode(self):
         meta_tag = soup.find("meta", {"name": "color-scheme"})
         self.assertIsNotNone(meta_tag)
         self.assertEqual(meta_tag["content"], "light")
+
+
+class FormattedfieldTagTestCase(SimpleTestCase):
+    def render_template(self, template, field_name="title", **context):
+        class BasicForm(forms.Form):
+            title = forms.CharField()
+
+        form = BasicForm(data={})
+
+        html = Template("{% load wagtailadmin_tags %}" + template).render(
+            Context(
+                {
+                    "field": form[field_name],
+                    **context,
+                }
+            )
+        )
+
+        # Parse the rendered HTML for the first DOM element in the output
+        return BeautifulSoup(html, "html.parser").find()
```

Comment:
Make the test class inherit from `WagtailTestUtils` (in addition to `SimpleTestCase`) and use `self.get_soup` instead

```suggestion
        return self.get_soup(html).find()
```